src/clients/customer/lib/ui/address_selection/address_selection_view.dart

```diff
@@ -19,26 +22,81 @@ class AddressSelectionView extends StatelessWidget with $AddressSelectionView {
     return ViewModelBuilder<AddressSelectionViewModel>.reactive(
       onModelReady: (model) => listenToFormUpdated(model),
       builder: (context, model, child) => Scaffold(
-        floatingActionButton: BoxButton(
-          title: 'Continue',
-          busy: model.isBusy,
-          disabled: !model.hasSelectedPlace,
-          onTap: () => model.selectAddressSuggestion(),
+        appBar: AppBar(
+          leading: IconButton(
+            icon: Icon(
+              Icons.close,
+              color: Colors.black,
+            ),
+            onPressed: () {},
+          ),
+          elevation: 0,
+          backgroundColor: Colors.transparent,
+          foregroundColor: kcPrimaryColor,
+        ),
+
+        /// Makes the button center
+        bottomNavigationBar: Visibility(
+          visible: model.hasSelectedPlace,
+          child: Container(
+            margin: EdgeInsets.all(20.0),
+            child: BoxButton(
+              title: 'Continue',
+              busy: model.isBusy,
+              disabled: !model.hasSelectedPlace,
+              onTap: () => model.selectAddressSuggestion(),
+            ),
+          ),
         ),
         body: ListView(
+          padding: EdgeInsets.all(20.0),
           children: [
-            TextFormField(
-              decoration: InputDecoration(hintText: 'Enter your address'),
-              controller: addressController,
+            Container(
+                child: Column(
+              mainAxisSize: MainAxisSize.min,
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                BoxText.headingTwo("Find restaurants near you"),
+                verticalSpaceSmall,
+                //TODO: Add Align property to BoxText
+                Text(
```

Comment:
All text should be BoxText in the entire app. This will ensure 100% ease of styling refactors. You can update the BoxText constructors to take in the alignment.